app/src/main/java/org/p2p/wallet/swap/jupiter/domain/model/SwapTokenModel.kt

```diff
@@ -0,0 +1,27 @@
+package org.p2p.wallet.swap.jupiter.domain.model
+
+import org.p2p.core.token.Token
+import org.p2p.wallet.swap.jupiter.repository.model.JupiterSwapToken
+import org.p2p.wallet.utils.Base58String
+import org.p2p.wallet.utils.toBase58Instance
+
+sealed interface SwapTokenModel {
+
+    val mintAddress: Base58String
+    val decimals: Int
+
+    data class UserToken(
+        val token: Token.Active,
+    ) : SwapTokenModel {
+        override val decimals: Int = token.decimals
+        override val mintAddress: Base58String = token.mintAddress.toBase58Instance()
+    }
+
+    data class JupiterToken(
+        val token: JupiterSwapToken,
```

Comment:
token -> details

Reply:
all the SwapTokens are JupiterToken
we can use our existent naming like Active / Other

Reply:
SwapTokenModel is not only JupiterToken
It's wrapper over Token.Active as user token with balance information 
and wrapper over JupiterSwapToken